aten/src/ATen/NamedTensorUtils.cpp

```diff
@@ -389,19 +389,17 @@ std::vector<Dimname> propagate_names_for_addmv(
   return unify_from_right(mv_outnames, bias.names());
 }
 
-void propagate_names_for_addmm(
-    Tensor& result,
+std::vector<Dimname> propagate_names_for_addmm(
```

Comment:
This is a little bit of a pessimization but not by much since the internals of this functional already always allocated a vector.

Reply:
seems already discussed in: https://github.com/pytorch/pytorch/pull/55746/files#r613660482. I can create a separate PR for that